CONTRIBUTING.md

```diff
@@ -150,8 +150,8 @@ For content changes, make sure that you:
   W3C service to [creating diff between HTML pages](https://services.w3.org/htmldiff)
   can also be used.
 * [ ] Review the content for technical accuracy.
-* [ ] Review the entire PR using the [localization
-  checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
+* [ ] Review the content using the [translation guide for
+  writers](https://github.com/github/docs/blob/main/contributing/translations/for-writers.md)
```

Comment:
https://github.com/github/docs/blob/main/contributing/translations/for-writers.md
is a 404
> The main branch of docs does not contain the path contributing/translations/for-writers.md.

Reply:
Good catch. It must've moved without redirecting. Updated the URL in https://github.com/solid/specification/commit/a7b07b77dfdaffddc55c900455203029f5fa408a